README.rst

```diff
@@ -153,12 +122,21 @@ Behavior
    causes nasty crashes by making it available through mass storage
    after the crash. A reset (the button) is needed after its fixed to
    get back into normal mode.
+-  RGB status LED
+-  Auto-reload after file write over mass storage. (Disable with
+   ``samd.disable_autoreload()``)
+-  Wait state after boot and main run, before REPL.
+-  Main is one of these: ``code.txt``, ``code.py``, ``main.py``,
+   ``main.txt``
+-  Boot is one of these: ``settings.txt``, ``settings.py``, ``boot.py``,
+   ``boot.txt``
 
 API
 ~~~
 
 -  Unified hardware APIs: `audioio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/audioio/__init__.html>`_, `analogio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/analogio/__init__.html>`_, `bleio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/bleio/__init__.html>`_, `busio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/busio/__init__.html>`_, `digitalio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/digitalio/__init__.html>`_, `pulseio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/pulseio/__init__.html>`_, `touchio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/touchio/__init__.html>`_, `microcontroller <https://circuitpython.readthedocs.io/en/latest/shared-bindings/microcontroller/__init__.html>`_, `board <https://circuitpython.readthedocs.io/en/latest/shared-bindings/board/__init__.html>`_, `bitbangio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/bitbangio/__init__.html>`_
```

Comment:
Do you want to add more here? `rotaryio`, etc.

Reply:
Removed the list and added a link instead.